src/OrbitGl/App.h

```diff
@@ -627,8 +629,9 @@ class OrbitApp final : public DataViewFactory,
   std::shared_ptr<SamplingReport> sampling_report_;
   std::shared_ptr<SamplingReport> selection_report_ = nullptr;
 
-  absl::flat_hash_map<std::pair<std::string, std::string>,
-                      orbit_base::Future<ErrorMessageOr<std::filesystem::path>>>
+  using ModuleLoadOperation =
+      std::pair<orbit_base::StopSource, orbit_base::Future<ErrorMessageOr<std::filesystem::path>>>;
```

Comment:
I think this should be a struct instead of a pair.

Reply:
done